### PR TITLE
DHE 159 Leeloo using Korben API to write to CDMS

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: leeloo/korben/connector.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import requests
+
+
+class Connector:
+
+    def __init__(self, table_name):
+        self.table_name = table_name
+        self.base_url = '{host}:{port}'.format(host=settings.KORBEN_HOST, port=settings.KORBEN_PORT)
+
+    def post(self, data, update=False):
```
Comment (bmcorser) [inline leeloo/korben/connector.py]: It's all post, just write seperate `update` and `create` methods.

File: leeloo/core/mixins.py [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+        korben_connector = Connector(table_name=self._meta.db_table)
+        korben_data = self._convert_model_to_korben_format()
+        korben_response = korben_connector.post(data=korben_data, update=update)
+
+        if korben_response:
+            self._map_korben_response_to_model_instance()
+            super().save(*args, **kwargs)
+
+        # update ES
+        save_model(self, update=update)
```
Comment (sdonk) [inline leeloo/core/mixins.py]: Yeah, I already moved that.

File: leeloo/korben/connector.py (hunk 1 above)
Comment (sdonk) [inline leeloo/korben/connector.py]: Nope, in that way I can use the update flag in the save() method and not have an if

**Comment by sdonk** [comment]
This PR is a stub, don't comment on it yet. I opened it only to show the code around

**Pull request closed without merging**